go.mod

```diff
@@ -301,9 +303,9 @@ replace (
 	github.com/gorilla/mux => github.com/gorilla/mux v1.7.0
 	github.com/gorilla/websocket => github.com/gorilla/websocket v1.4.0
 	github.com/gregjones/httpcache => github.com/gregjones/httpcache v0.0.0-20170728041850-787624de3eb7
-	github.com/grpc-ecosystem/go-grpc-middleware => github.com/grpc-ecosystem/go-grpc-middleware v0.0.0-20190222133341-cfaf5686ec79
+	github.com/grpc-ecosystem/go-grpc-middleware => github.com/grpc-ecosystem/go-grpc-middleware v1.0.1-0.20190118093823-f849b5445de4
```

Comment:
this backlevels us one commit, but that commit did not actually change any go files:
```
git diff cfaf5686ec79 f849b5445de4 --numstat
1       1       .travis.yml
26      32      Gopkg.lock
```